stumpy/gpu_stump.py

```diff
@@ -163,22 +171,17 @@ def _compute_and_update_PI_kernel(
         if math.isinf(M_T[j]) or math.isinf(μ_Q[i]):
             p_norm = np.inf
         else:
```

Comment:
I am wondering if this needs to be indented or if it is possible just to do:

```
        if math.isinf(M_T[j]) or math.isinf(μ_Q[i]):
            p_norm = np.inf
        elif T_subseq_isconstant[j] and Q_subseq_isconstant[i]:
            p_norm = 0
        elif T_subseq_isconstant[j] or Q_subseq_isconstant[i]:
            p_norm = m
        else:
            denom = m * σ_Q[i] * Σ_T[j]
```

Would you mind checking?

Reply:
Thank your for catching this! I will check it.